include/onnxruntime/core/framework/ort_event.h

```diff
@@ -11,8 +15,11 @@ struct OnnxRuntimeEvent {
   bool finished = false;
   OnnxRuntimeEvent() = default;
 
- private:
   ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE(OnnxRuntimeEvent);
 };
 
 using ORT_EVENT = OnnxRuntimeEvent*;
+
```

Comment:
Do we need this 'using' statement? Aliases that hide that something is a pointer often make it harder to read the code.

Reply:
Will. I'll add another implementation of this ORT_EVENT, that is just a win32 handle.

Reply:
Updated.

Reply:
Consider using the Notification class present in the framework folder if it serves the same purpose. It was copied from tensorflow.

Reply:
Hi @pranavsharma  resolved.

Reply:
Thanks. Curious, why does windows need a different implementation? Wouldn't Notification work for windows as well?

Reply:
1. faster
2. please see the comments in this file.

Reply:
I'll consider your suggestion.